content/internal/policy-coding-standards.md

```diff
@@ -265,6 +282,16 @@ from mypkg.sibling import example
 import sys, os
 ```
 
+### Method chaining
+
+- Method chaining is simply being able to add a second function to whatever the first function returns.
```

Comment:
Probably better to use "method" rather than "function" because these are two different things in Python.